bin/generate_label.py

```diff
@@ -26,8 +36,29 @@ def main(stem, h5ad):
         # md[sample_id] = adata.uns["spatial"][i]["scalefactors"]
     # adata.obs["Y"] = adata.obsm["spatial"][:, 0]
     # adata.obs["X"] = adata.obsm["spatial"][:, 1]
-    print(adata.uns["spatial"])
-    print(adata.obsm["spatial"])
+    #----------------------------------------------
+
+    # print(adata.uns["spatial"][sample_id]["scalefactors"])
+
+    spot_diameter_fullres = adata.uns["spatial"][sample_id]["scalefactors"]["spot_diameter_fullres"]
```

Comment:
I'd say no. Just enforce it to be present, and just fail otherwise. Since it varies even between sections. Giving a default value will for sure lead to complains...